great_expectations/datasource/fluent/config_str.py

```diff
@@ -39,6 +39,7 @@ def get_config_value(self, config_provider: _ConfigurationProvider) -> str:
         Resolve the config template string to its string value according to the passed
         _ConfigurationProvider.
         """
+        LOGGER.info(f"Substituting '{self}'")
```

Comment:
Note this logs the config template string, not the potentially sensitive substituted value.